src/threadColors.js

```diff
@@ -1,22 +1,39 @@
 "use strict";
 
-module.exports = function(defaultFuncs, api, ctx) {
+module.exports = function(_defaultFuncs, _api, _ctx) {
   // Currently the only colors that can be passed to api.changeThreadColor(); may change if Facebook adds more
   return {
-    MessengerBlue: null,
-    Viking: "#44bec7",
-    GoldenPoppy: "#ffc300",
-    RadicalRed: "#fa3c4c",
-    Shocking: "#d696bb",
-    PictonBlue: "#6699cc",
-    FreeSpeechGreen: "#13cf13",
-    Pumpkin: "#ff7e29",
-    LightCoral: "#e68585",
-    MediumSlateBlue: "#7646ff",
-    DeepSkyBlue: "#20cef5",
-    Fern: "#67b868",
-    Cameo: "#d4a88c",
-    BrilliantRose: "#ff5ca1",
-    BilobaFlower: "#a695c7"
+    ////MessengerBlue: null,
+    ////Viking: "#44bec7",
+    ////GoldenPoppy: "#ffc300",
+    ////RadicalRed: "#fa3c4c",
+    ////Shocking: "#d696bb",
+    ////PictonBlue: "#6699cc",
+    ////FreeSpeechGreen: "#13cf13",
+    ////Pumpkin: "#ff7e29",
+    ////LightCoral: "#e68585",
+    ////MediumSlateBlue: "#7646ff",
+    ////DeepSkyBlue: "#20cef5",
+    ////Fern: "#67b868",
+    ////Cameo: "#d4a88c",
+    ////BrilliantRose: "#ff5ca1",
+    ////BilobaFlower: "#a695c7"
```

Comment:
Are these the same colors with different names? This is a really breaking change...

Everything else looks great

Reply:
umm, those are some old color which have been commented. also facebook uses theme id now, so those old colors can't be used anymore. 
and no, we can't keep them.

Reply:
I understand that the api has changed, but clearly some of these colors must be similar `MessengerBlue` vs `DefaultBlue` for example.. I remember the original names came from the messenger code somewhere, where did you get the new names?
I'm sure you understand that this change break anyone who currently uses this api

Reply:
I get these names from this
![image](https://user-images.githubusercontent.com/29724654/73587113-d0d65e80-44e9-11ea-9d70-1bb0276bcd33.png)
and yes, I understand that this change will break compatibility.

Reply:
@lequanglam you should put new ids in place of old hex colours (instead of commenting them). This way we can maintain backward compatibility.

Reply:
@ravkr will try to make it backward-compatible, but i'm currently busy at listenMqtt's bugs.